mod-director-launchwrapper/src/main/java/net/jan/moddirector/launchwrapper/forge/ForgeLateLoader.java

```diff
@@ -335,6 +335,17 @@ private Method getMethod(String[] possibleNames, Class<?> targetClass, Class<?>.
 
     private void handle(InstalledMod mod) {
         Path injectedFile = mod.getFile();
+        
+        switch(mod.getInject()) {
+	        case 1:
+	        	break;
+	        case 2:
+	        	return;
+	        case 0:
+	        default:
+	            if (!mod.getFile().toString().contains("mods")) return;
+	            break;
+        }
```

Comment:
I'm not really a fan of this logic. I think the better option would be to just check here if the option is `true` (see other review comments on the tristate issue).

Whether to inject the mod if `inject` is not given should simply depend on whether a custom directory is given. If the user adds `mods` as a custom directory, we might want to emit a warning (or just not, since there is really no need to override the directory in case you just want the mod to be a normal mod and injected).

Reply:
Did exactly this.